next.config.mjs

```diff
@@ -22,11 +22,14 @@ const nextConfig = {
     }
 
     return config;
-  }
+  },
+  'trailingSlash': true,
```

Comment:
Add a comment above on why this is added in next config.

Reply:
@sambhavgupta0705 Can we please remove the this `trainlingSlash` configuration? It was in the i18n setup article, but I think it does not have any relevance with the issue.